pkg/flare/archive.go

```diff
@@ -726,9 +734,13 @@ func walkConfigFilePaths(tempDir, hostname string, confSearchPaths SearchPaths,
 					return err
 				}
 
-				if permsInfos != nil {
+				if len(permsInfos) != 0 {
```

Comment:
Can you explain why this code should not be called when `len(permsInfos) == 0` (previously it was when `permsInfos` was nil)

Reply:
Yes, because on windows, permsInfo stays empty. That check is meant so the code doesn’t get executed on windows. nil was wrong, because the map gets initialized. So len == 0 checks more accurately for an empty map on windows. Should I leave a comment in code?